relay_rpc/src/rpc/tests.rs

```diff
@@ -639,13 +592,13 @@ fn validation() {
     };
     assert_eq!(request.validate(), Ok(()));
 
-    // Batch receive: empty list.
+    // Batch receive: empty list.PayloadError
```

Comment:
:smile: multi line select?

Reply:
Initially I commented these tests entirely to update them later. Then, I renamed `ValidationError` to `PayloadError` using rust analyzer. And something weird happend: apparently it tried to rename identifiers in the commented block, but failed leaving the block completely messed up, with these new identifiers scattered randomly throughout the block.